tests/test_tasks.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+import requests_mock
+
+from udata.core.dataset.factories import DatasetFactory
+
+from udata_transport.tasks import map_transport_datasets
+
+MOCK_URL = 'http://reco.net'
+
+
+@pytest.fixture
+def mock_response():
+    return [
+        {
+            "datagouv_id": "61fd29da29ea95c7bc0e1211",
```

Comment:
`id` and `datagouv_id` are the same?

Reply:
seems but in doubt I use the `datagouv_id` returned by transport API